### PR TITLE
Add San Jose file

### PR DESCRIPTION
**Pull request by mleacct**
this is the beginning of San Jose file.
Please provide feedback on
- what you would like to do when visiting San Jose
- @patricia5859 are you looking for Italian restaurant during your visit?
- @medianick Hi Nick. could you add some comment on my pull request?

### CODE REVIEW

**Comment by shinton** [comment]
Maybe you could add a link to a picture of the famous/controversial statue downtown? 😀

File: mleacct-sanjoseca.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# San Jose, CA
+
+### Restaurants
+
+### Local Attractions
+- Great America Theme Park
```
Comment (diarmaidm) [inline mleacct-sanjoseca.md]: Is that something like 6 Flags?

Comment (mleacct) [inline mleacct-sanjoseca.md]: Add to attractions.  Thanks

Comment (mleacct) [inline mleacct-sanjoseca.md]: @shinton sorry I don't know about "famous/controversial statue downtown" of San Jose.

Comment (shinton) [inline mleacct-sanjoseca.md]: Hehe! [Quetzy](http://www.sanjose.com/underbelly/unbelly/Sanjose/Quetzy/quetzy.html)

**Comment by NatashaE** [comment]
Looking good! :shipit: 🚢 👍

**Comment by diarmaidm** [comment]
:+1:

**Pull request merged**